Google.Cloud.EntityFrameworkCore.Spanner/Storage/Internal/SpannerDataReaderWithChecksum.cs

```diff
@@ -65,6 +65,18 @@ internal sealed class SpannerDataReaderWithChecksum : DbDataReader, IRetriableSt
 
         private SpannerDataReader _spannerDataReader;
 
+        public override void Close()
```

Comment:
Should we add some unit test cases to verify if _spannerDataReader is closed/disposed or not ?

Reply:
Good point, that will prevent any regressions in the future if someone by accident removes the behavior. Added a verification to one of the existing tests.